### PR TITLE
Add variant A and B links for PopularTasks AB test

### PR DESCRIPTION
**Pull request by hannako**
## What

Add URL's and titles for variants A and B for the upcoming PopularTasks AB test

**This PR will be easier to review commit by commit**

- [Add variant A links](https://github.com/alphagov/collections/pull/3744/commits/ccfaf7163995f67ac5cbb51aec92cb87bd91b2b7) commit.
Variant A is most searched for destination pages from the browse page. The data has come from [this sheet](https://docs.google.com/spreadsheets/d/1K76hX8e02y2dHjcZrS_nqKNY94hk4YBP5UxpTUCvEzc/edit?usp=sharing)
- [Add variant B links](https://github.com/alphagov/collections/pull/3744/commits/ec83e875d2502f0a24d9ad790be4af0a7d1f34a3) commit.
Variant B is most viewed content tagged to any browse subtopic. The data has come from [this sheet](https://docs.google.com/spreadsheets/d/1RwCIut30zAnZaZ4FFItKrvpeowTn_M4Dt5WMiH3ZhlA/edit?usp=sharing)

## How to test
- Install [Modheader](https://chromewebstore.google.com/detail/modheader-modify-http-hea/idgpnmonknjnojddfkpgkljpfnnfcklj?hl=en) or similar tool

- Set the following request header, and toggle variants A, B, C or Z

Screenshot 2024-08-21 at 21 26 49

## Review apps

- [abroad](https://collections-pr-3744.herokuapp.com/browse/abroad)
- [benefits](https://collections-pr-3744.herokuapp.com/browse/benefits)
- [births-deaths-marriages](https://collections-pr-3744.herokuapp.com/browse/births-deaths-marriages)
- [business](https://collections-pr-3744.herokuapp.com/browse/business)
- [childcare-parenting](https://collections-pr-3744.herokuapp.com/browse/childcare-parenting)
- [citizenship](https://collections-pr-3744.herokuapp.com/browse/citizenship)
- [disabilities](https://collections-pr-3744.herokuapp.com/browse/disabilities) 
- [driving](https://collections-pr-3744.herokuapp.com/browse/driving) 
- [education](https://collections-pr-3744.herokuapp.com/browse/education)
- [employing-people](https://collections-pr-3744.herokuapp.com/browse/employing-people) 
- [environment-countryside](https://collections-pr-3744.herokuapp.com/browse/environment-countryside)
- [housing-local-services](https://collections-pr-3744.herokuapp.com/browse/housing-local-services)
- [justice](https://collections-pr-3744.herokuapp.com/browse/justice) 
- [tax](https://collections-pr-3744.herokuapp.com/browse/tax) 
- [visas-immigration](https://collections-pr-3744.herokuapp.com/browse/visas-immigration) 
- [working](https://collections-pr-3744.herokuapp.com/browse/working) 

## Screenshot example (for browse benefits)

Control|Variant A|Variant B
---|---|---
Screenshot 2024-08-21 at 21 31 13|Screenshot 2024-08-21 at 21 30 48|Screenshot 2024-08-21 at 21 31 01

Related work:
- https://github.com/alphagov/govuk-fastly/pull/93
Trello card: https://trello.com/c/cy7wYrdz/2803-add-variant-a-and-variant-b-links-to-collections

### CODE REVIEW

**Review by unoduetre — APPROVED** [review]
Looks good to me

File: config/locales/en/browse.yml [hunk 1]
```diff
@@ -2,16 +2,54 @@
...
             url: /check-benefits-financial-support
         variant_a:
-          - title:
-            url:
+          - title: Universal Credit
+            url: /universal-credit
+          - title: Sign in to your Universal Credit account
+            url: /sign-in-universal-credit
+          - title: Child Benefit
+            url: /child-benefit/how-to-claim
```
Comment (unoduetre) [inline config/locales/en/browse.yml:30]: In the spreadsheet it's /child-benefit

**Review by unoduetre — CHANGES_REQUESTED** [review]

File: config/locales/en/browse.yml (hunk 1 above)
Comment (hannako) [reply config/locales/en/browse.yml:30]: Oh good spot Mat - thank you so much 🕵️  I will amend

Comment (hannako) [reply config/locales/en/browse.yml]: Done

**Review by unoduetre — APPROVED** [review]
I also did some checks with modified headers.

Approved.

**Pull request merged**